plugins/woocommerce/includes/wc-update-functions.php

```diff
@@ -2460,7 +2460,7 @@ function wc_update_700_remove_download_log_fk() {
  * Remove the transient data for recommended marketing extensions.
  */
 function wc_update_700_remove_recommended_marketing_plugins_transient() {
-	delete_transient( MarketingSpecs::RECOMMENDED_PLUGINS_TRANSIENT );
+	delete_transient( 'wc_marketing_recommended_plugins' );
```

Comment:
This update function can be deleted because https://github.com/woocommerce/woocommerce/pull/43162 will be introduced. No need to have two update functions doing the same thing in case someone updates from pre-7.0

Reply:
I'm not really sure whether we should do this. 🤔 

Technically, we can delete the function, and things would work as expected.

However, in the way I see it, this `plugins/woocommerce/includes/wc-update-functions.php` file also acts like a "changelog" of database changes. We can look into all the `wc_update_XXX` (where XXX is the 3 digit WooCommerce version number) and know what happened in the database migration during each update. We can even go all the way to version 2.0 from 6 years ago.

If we delete the function, then it feels like we are deleting a record in the "changelog".

To be on the safe side, I'm leaning towards keeping it.

What do you think?

Reply:
It is redundant this is why I don't like it but I understand the reason behind having it pemanent.